modules/actions/pool/staking/sync-aura-staking.ts

```diff
@@ -0,0 +1,77 @@
+import { Chain, PrismaPoolStakingType } from '@prisma/client';
+import { AuraSubgraphService } from '../../../sources/subgraphs/aura/aura.service';
+import { prisma } from '../../../../prisma/prisma-client';
+import { prismaBulkExecuteOperations } from '../../../../prisma/prisma-util';
+
+export const syncAuraStakingForPools = async (
+    chain: Chain,
+    auraSubgraphService: AuraSubgraphService,
+): Promise<void> => {
+    const auraGauges = await auraSubgraphService.getAllPools([chain]);
+    const pools = await prisma.prismaPool.findMany({
+        where: {
+            chain: chain,
+            address: { in: auraGauges.map((gauge) => gauge.lpToken.address) },
+        },
+        include: { staking: true },
+    });
+
+    const operations: any[] = [];
+
+    for (const auraGauge of auraGauges) {
+        const pool = pools.find((pool) => pool.address === auraGauge.lpToken.address);
+
+        if (!pool) {
+            continue;
+        }
+
+        const stakingId = auraGauge.id + '-aura';
+
+        const dbStaking = pool.staking.find((staking) => staking.chain === chain && staking.id === stakingId);
+
+        if (!dbStaking) {
+            operations.push(
+                prisma.prismaPoolStaking.upsert({
```

Comment:
sorry, i was here many times already, but always forget what's the "prismaPoolStaking", is it a gauge? and maybe we can simplify related tables, eg: "prismaPoolStakingAura"? I looks like they repeat a lot and one table with type and json can replace all 5 tables. eg:

```
incentives (or incentive_contracts)  {
  address, type, pool_id, chain, status, data (json – used to calculate the APR – stored separately?)
}
```